Configure local play bar

MyNameIsJonathan opened the pull request.

angeliquemari commented on database/Cassandra/schema.cql:
```diff
@@ -0,0 +1,21 @@
+CREATE TABLE likes (
+   id int,
+   song_id int,
+   user_id int,
```
are you planning to make a users table? if not, it might make more sense for this to be `user` or `username` rather than an id

MyNameIsJonathan replied:
I changed user_id to userName, because all I'll need here is if the current user has liked the current song

angeliquemari commented on database/data/schema.sql:
```diff
@@ -1,37 +1,27 @@
-    REFERENCES songs(id)
-    ON UPDATE CASCADE,
-  UNIQUE KEY (position)
+  time_stamp INT DEFAULT 0,
```
what does timestamp refer to? since you've made it an integer type and not timestamp type

MyNameIsJonathan replied:
Timestamp was a vestige of Lily's implementation -- I removed it!

angeliquemari commented on database/postgreSQL/schema.sql:
```diff
@@ -0,0 +1,25 @@
+CREATE TABLE likes (
+  id INT NOT NULL AUTO_INCREMENT,
+  user_id INT,
+  song_id INT,
```
you might want to specify this as a foreign key constraint with certain cascade behaviour e.g. if a song is deleted the likes associated with it would also be deleted

angeliquemari commented on client/metaHelpers.js:
```diff
@@ -70,29 +71,19 @@ const metaHelpers = {
+  like(song_id, is_liked) {
     const {upNext} = this.state;
-    //  Post to the "http://localhost/like:songId" route to toggle like status
+    //  Post to the "http://localhost:3000/likes:song_id" route to toggle like status
```
detail: `:3000/likes/:song_id`

MyNameIsJonathan commented:
I have followed Angelique's recommendations and have:
changed case formatting to all camelCase
Altered table column names to better reflect usage instances
Removed unnecessary code carried over from FEC
